src/main/java/com/twilio/rest/api/v2010/account/CallReader.java

```diff
@@ -59,6 +60,16 @@ public CallReader setTo(final com.twilio.type.PhoneNumber to) {
         return this;
     }
 
+    /**
+     * Only show calls to this phone number or Client identifier.
+     * 
+     * @param to Phone number or Client identifier to filter `to` on
+     * @return this
+     */
+    public CallReader setTo(final String to) {
```

Comment:
Are these new filter parameters or are adding these promotions?

Reply:
From adding the promotion these got generated automatically.